pulumi/backuprunner/.gitignore

```diff
@@ -0,0 +1,129 @@
+# Project specific
```

Comment:
think at this point it would just make sense to consolidate all `.gitignores` to a single top level file